src/NUnitFramework/framework/Internal/Randomizer.cs

```diff
@@ -611,6 +611,25 @@ public decimal NextDecimal(decimal min, decimal max)
             return unchecked(raw % range + min);
         }
 
+        #endregion 
+
+        #region Guid
+
+        /// <summary>
+        /// Returns a random <see cref="Guid"/>.
```

Comment:
Apparently since Windows 2000, `CoCreateGuid` has only ever been observed generating type 4 guids. But we shouldn't make it seem like that's documented by Microsoft.